crypto/ct/ct_vfy.c

```diff
@@ -113,6 +113,10 @@ int SCT_CTX_verify(const SCT_CTX *sctx, const SCT *sct)
         CTerr(CT_F_SCT_CTX_VERIFY, CT_R_SCT_LOG_ID_MISMATCH);
         return 0;
     }
+    if (sct->timestamp > sctx->epoch_time_in_ms) {
```

Comment:
This, especially given that time is in ms, does not leave room for any clock drift.  How fresh an SCT might reasonably be encountered in the field?  With LE automation issuing certs, might one not expect that certificates could be minted and deployed in less than 1 second?